Doc/library/json.rst

```diff
@@ -262,6 +262,10 @@ Basic Usage
    .. versionchanged:: 3.6
       All optional parameters are now :ref:`keyword-only <keyword-only_parameter>`.
 
+   .. versionchanged:: 3.6
+      *fp* can now be a :term:`binary file`. The input encoding should be
+      UTF-8, UTF-16 or UTF-32.
```

Comment:
IMHO the feature should also be documented in the body of the function documentation, not also in this versionchanged note.

Reply:
agreed, added!